client/wfm/src/result/WorkOrderResult.ts

```diff
@@ -0,0 +1,11 @@
+import {StepResult} from './StepResult';
+export interface WorkOrderResult {
+  id: string;
+  status: string;
+  nextStepIndex: number;
```

Comment:
I think the solution should only move to the next step by default, and have specialized steps for branching.

Right now I think the most we can do with this field is move it to another entity like `workorder.nextStepIndex`, though I'm not sure it's worth it.
Bigger changes in this logic should be in another ticket I guess.